benchmarks/index.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+const fs = require("fs");
+const Benchmark = require("benchmark");
+const benchmarks = require("beautify-benchmark");
+const suite = new Benchmark.Suite();
+
+const actions = require("../lib/actions");
+
+const badBundleFixtureRoot = require
+  .resolve("inspectpack-test-fixtures")
+  .replace("/index.js", "");
```

Comment:
The better pattern for this that is **always** correct is:

```js
const MODULE_NAME_ROOT = path.dirname(require.resolve("MODULE_NAME/package.json"));
```

Because **every** npm module must have `package.json`.